ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -31,7 +31,7 @@ def test_vector_to_skew_symmetric_matrix(
     fw,
     a,
 ):
-    if "float16" or "int8" in input_dtype:
+    if "float16" in input_dtype or "int8" in input_dtype:
```

Comment:
we should actually remove this `if` statement entirely, it should be handled correctly by the helper.

Reply:
Oh I cooperated with Darshan to fix this submodule, and I was responsible for the last 12 tests (From line 536 test_tensordot to the bottom of the file)

Reply:
Btw, I modified line 34 since I noticed that many of the tests were actually not passed.
eg. In this if block, 
`   
 if "float16" in input_dtype or "int8" in input_dtype:
        return

`
it will go to the "return"

Reply:
> we should actually remove this `if` statement entirely, it should be handled correctly by the helper.

Hi @djl11, I've removed this and fixed the vector_to_skew_symmetric_matrix test function on another branch where I've worked on the remaining 12 functions of linalg submodule. I'll request a PR shortly.